ops/terraform/services/.terraform-version

```diff
@@ -0,0 +1 @@
+1.1.9
```

Comment:
I assume by the .terraform-version naming you are suggesting the use of tfenv?

Reply:
That's right! Obviously, we're still firmly in a mode of v0.12.31 for the existing terraform, so we'll need to support both old and new for a time.